### DISCUSSION TITLE
Help Needed: WordPress Site Uploaded to GitHub - CSS and JS Not Functioning

### DISCUSSION THREAD

**Discussion by Kristijan-Peshevski** [community/community#65226] [New to GitHub] [answered]
Hello everyone,

I'm facing an issue with a WordPress website that I've uploaded to GitHub. Here's the process I followed:

Created a repository using the GitHub app.
Added WordPress to the repository.
Developed the website locally using XAMPP.
Utilized Simply Static to convert the site to a static WordPress version.
Committed the changes and pushed them to the 'gh-pages' branch.
However, when I access the site on GitHub Pages, only the HTML content displays, while the CSS and JavaScript don't seem to be functioning correctly.

I've already checked file paths, made sure references are relative, and reviewed my repository settings, but the issue persists.

Any suggestions on how to troubleshoot and resolve this problem would be greatly appreciated. Thank you in advance for your assistance!

**Comment by queenofcorgis** [comment]
Hi there @Kristijan-Peshevski , 

Thanks for posting! Are you able to share the repository in question? That may help the community troubleshoot.

**Reply by Kristijan-Peshevski** [reply]
This is the website link: https://kristijan-peshevski.github.io/WordPress-Website/
And this is the repository: https://github.com/Kristijan-Peshevski/WordPress-Website

I created a new repository and pushed my WordPress files there and this time it looks a little different but still it doesn't show the images and the navigation menu doesn't work.

**Comment by Ba3a-G** [comment]
There is some issue with your URL scheme. 

![image](https://github.com/community/community/assets/52352294/fbb92f6a-808b-46c9-8436-d81ccf2b070f)
![image](https://github.com/community/community/assets/52352294/50ca18c9-d2e5-4e9f-974a-0b4b59353b83)

Not sure what's causing it. I suggest you re-deploy. If it still doesn't work, I'll fork your repo and try to find out the issue.

**Reply by Kristijan-Peshevski** [reply]
I made new WordPress Websites and tried uploading them to GitHub Pages but still, it doesn't work, I tried everything.

**Comment by favianaldo** [comment, marked as answer]
try this,
in index.html you have problem in href: httphttps
please change it to https 

line 33, 52, 53, 54, 93, etc

i repaired manual
![trou](https://github.com/community/community/assets/60067744/0db0666a-8bbd-40fe-93a0-3a7e09d33c25)

**Comment by JaniVilen** [comment]
When a [WordPress](https://sf2apk.com/shadow-fight-2-mod-apk/) site is uploaded to GitHub and its CSS and JS are not functioning, it's typically due to relative file paths. GitHub may alter URLs. Update paths to reflect the GitHub structure or use absolute URLs. Ensure file permissions and Gitignore settings are correct. Test on a local server.
